build.gradle

```diff
@@ -39,27 +42,67 @@ compileJava {
 
 
 task kb_sdk_plusScript {
-	dependsOn compileJava
+	// creates a kb-sdk script based on the jars in their default locations
+	dependsOn jar
 	doLast {
 		def dependencies = configurations.runtimeClasspath.collect { File file ->
 			file.absolutePath
 		}
 	
-		def classpath = dependencies.join(':')
+		def buildjar = "$buildDir/libs/${project.name}.jar"
+		def classpath = buildjar + ":" + dependencies.join(':')
 	
 		def scriptContent = """#!/bin/sh
 
 CLASSPATH=$classpath
-MOD=us.kbase.mobu.ModuleBuilder
 
-java -cp $buildDir/classes/java/main:$buildDir/resources/main:\$CLASSPATH \$MOD \$@
+java -cp \$CLASSPATH us.kbase.mobu.ModuleBuilder \$@
 """
 		def outfile = "$buildDir/kb-sdk"
 		file(outfile).text = scriptContent
 		file(outfile).setExecutable(true)
 	}
 }
 
+task prepareRunnableDir {
+	// like kb_sdk_plusScript but localizes all the jars and the script
+	// into a directory for easy copying
+	// Note requires bash vs just sh
+	def runnableDir = file("$buildDir/runnable")
+	def runnerScript = new File(runnableDir, "kb-sdk")
+
+	doFirst {
+		// Clean and recreate runnable dir
+		delete runnableDir
+		mkdir runnableDir
+
+		// Copy all runtime jars
+		configurations.runtimeClasspath.each { File dep ->
+			copy {
+				from dep
+				into runnableDir
+			}
+		}
+
+		// Copy the project's jar
+		def jarFile = file("$buildDir/libs/${project.name}.jar")
```

Comment:
Should this task have a `dependsOn jar` line, too?

Reply:
Yes, I caught that while making the next pr so it's fixed there